src/components/CandidateAvatar/CandidateAvatar.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from '../../packages/bem';
+import styles from './CandidateAvatar.scss';
+
+const { block, elem } = bem({
+    name: 'CandidateAvatar',
+    classnames: styles,
+    propsToMods: ['context']
+});
+
+const context = percentage => {
+    if (percentage <= 33) {
+        return 'bad';
+    }
+    if (percentage <= 66) {
+        return 'warning';
+    }
+    return 'good';
+};
+
+const CandidateAvatar = props => {
+    const { imageUrl, matchPercentage, showPercentageOnHover, size, ...rest } = props;
+
+    const fixedSize = Math.round(Math.max(32, size));
+    const radius = fixedSize / 2 - 2;
+    const percentage = Math.max(0, Math.min(100, matchPercentage));
+    const circumference = 2 * radius * Math.PI;
+    const strokeDasharray = `${(percentage * circumference) / 100} 999`;
+
+    return (
+        <div
+            {...rest}
+            {...block(props)}
+            style={{
+                width: fixedSize,
+                height: fixedSize
+            }}
+        >
+            <div
+                {...elem('image', props)}
+                style={{
+                    backgroundImage: `url(${imageUrl})`
+                }}
+            >
+                {!!showPercentageOnHover && (
+                    <div {...elem('percentage', props)}>{`${percentage}%`}</div>
+                )}
+            </div>
+            <svg
+                {...elem('ring', props)}
+                style={{
+                    width: fixedSize,
+                    height: fixedSize
+                }}
+            >
+                <circle
+                    r={radius}
+                    cx={fixedSize / 2}
+                    cy={fixedSize / 2}
+                    strokeWidth="4"
+                    {...elem('circle', {
+                        ...props,
+                        context: context(percentage)
+                    })}
+                    style={{
+                        strokeDasharray
+                    }}
+                />
+            </svg>
+        </div>
+    );
+};
+
+CandidateAvatar.propTypes = {
+    imageUrl: PropTypes.string,
+    matchPercentage: PropTypes.number,
+    showPercentageOnHover: PropTypes.bool,
+    size: PropTypes.number
+};
+
+CandidateAvatar.defaultProps = {
+    imageUrl: null,
```

Comment:
I think we should add some default image not just leave it blank.

Reply:
We haven't yet figured out how to deal with static assets.. So I'm not sure how to include an image without linking to an external host

Reply:
good point. maybe can add to the statics ticket as a task to add an image here. so that it isn't forgotten.

Reply:
Done